etc/omero.properties

```diff
@@ -208,9 +208,9 @@ versions.omero-scripts-url=${versions.omero-pypi}
 ###
 
 ## Internal dependencies
-versions.omero-blitz=5.5.9
-versions.omero-common-test=5.5.8
-versions.omero-gateway=5.6.8
+versions.omero-blitz=5.5.10
+versions.omero-common-test=5.5.9
+versions.omero-gateway=5.6.9
 versions.omero-scripts=5.6.1
```

Comment:
Should this also be bumped to `version.omero-scripts=5.6.2`  which includes several bug fixes - see https://github.com/ome/omero-scripts/compare/v5.6.1...v5.6.2 ?

Reply:
yes. I thought i did bump it